### DISCUSSION TITLE
Keyframe Perspective and Warp

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#617] [Ideas]
If there's anything I can give a point to Procreate Dreams for, it's being able to keyframe perspective and warp transform. I hope this is in the next update, or any other future updates, because I realized that perspectives transform can do a "shear" transform, where one side of a transform is shifted to the side, giving the drawing a slanted look, which can be beneficial for rigged YouTube animators. Also, it'd be nice if it were possible to add perspective and warp transform to vector brushes and lines as well.

**Comment by y4r3l** [comment]
very important feature for amazing 3d effects
